src/wazuh_modules/wm_gcp.c

```diff
@@ -0,0 +1,263 @@
+/*
+ * Wazuh Module for Security Configuration Assessment
+ * Copyright (C) 2015-2019, Wazuh Inc.
+ * January 25, 2019.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "wmodules.h"
+#include "os_net/os_net.h"
+#include <sys/stat.h>
+#include "os_crypto/sha256/sha256_op.h"
+#include "shared.h"
+
+static void* wm_gcp_main(wm_gcp *gcp_config);                        // Module main function. It won't return
+void wm_gcp_run(wm_gcp *data);                                       // Running python script
+static void wm_gcp_destroy(wm_gcp *gcp_config);                      // Destroy data
+cJSON *wm_gcp_dump(const wm_gcp *gcp_config);                        // Read config
+
+/* Context definition */
+
+const wm_context WM_GCP_CONTEXT = {
+    GCP_WM_NAME,
+    (wm_routine)wm_gcp_main,
+    (wm_routine)(void *)wm_gcp_destroy,
+    (cJSON * (*)(const void *))wm_gcp_dump
+};
+
+// Module main function. It won't return
+void* wm_gcp_main(wm_gcp *data) {
+    time_t time_start;
+    time_t time_sleep = 0;
+
+    // If module is disabled, exit
+    if (data->enabled) {
+        mtinfo(WM_GCP_LOGTAG, "Module started");
+    } else {
+        mtinfo(WM_GCP_LOGTAG, "Module disabled. Exiting.");
+        pthread_exit(NULL);
+    }
+
+    if (!data->pull_on_start) {
```

Comment:
Please, add support to the other options to timing the module: `time`, `day`, `wday`. It should be consistent with them: https://documentation.wazuh.com/3.10/user-manual/reference/ossec-conf/sca.html#scheduling-options